pkg/noderesourcetopology/filter.go

```diff
@@ -143,3 +145,15 @@ func (tm *TopologyMatch) Filter(ctx context.Context, cycleState *framework.Cycle
 	}
 	return nil
 }
+
+// resourceFoundOnNode checks whether a given resource exist at the node level
+// and whether the given quantity is big enough
+func resourceFoundOnNode(wantResource v1.ResourceName, wantQuantity resource.Quantity, nodeInfo *framework.NodeInfo) bool {
```

Comment:
I'm not sure is it worth iterating every time, maybe just create a map in Filter, and pass then it instead of NodeInfo, WDYT?

Reply:
SGTM, I'll do that

Reply:
My mistake ```nodeInfo.Allocatable.ResourceList()``` is already a map so don't need to iterate at all.